bolt/lib/Profile/DataAggregator.cpp

```diff
@@ -903,24 +956,6 @@ DataAggregator::getFallthroughsInTrace(BinaryFunction &BF,
   if (!FromBB || !ToBB)
     return std::nullopt;
 
-  // Adjust FromBB if the first LBR is a return from the last instruction in
```

Comment:
For reference to aid in review the code being removed here comes from 5599c019117a